day_two_guide/topics/node_maintenance.adoc

```diff
@@ -260,9 +260,9 @@ is lost. The location of the data in the *{rhocp}* node is located in
 be configured:
 
 NOTE: If the XFS filesystem hosting that folder is mounted with the
-`gquota` option in the `/etc/fstab`
+`grpquota` option in the `/etc/fstab`
 * the matching security context contraint’s `fsGroup` type set to `MustRunAs`
```

Comment:
The structure of this list is confusing, and I'm not sure what you mean by "The using the ... configuration map."

Maybe

---
`grpquota` option in the `/etc/fstab` file:
* Set the matching security context contraint’s `fsGroup` type to `MustRunAs`.
* xref:../install_config/master_node_configuration.adoc#node-config-volume-config[Configure the volume] using the *_node-config-compute_* configuration map in the *openshift-node* project.
(and remove the example volumeconfig because it's not really helping here.)
---

Reply:
@kalexand-rh Looking around, I don't think this topic is being used at this time. I made your changes, but did not remove the example. I am not sure why it is there.
I did a search and found gquota and changed to grpquota per Seth's comment.

Reply:
Thanks, @mburke5678!
@bfallonf, do you know if this file should be used or removed? It looks like it was commented out of the day_two_guide/host_level_tasks topic.